src/main/java/frc/robot/subsystems/Pitch.java

```diff
@@ -201,6 +211,10 @@ public Optional<Rotation2d> getDesiredAngleToLock(Pose2d robotPose, Pose3d[] fie
     return Optional.of(desiredAngle);
   }
 
+  public boolean isPitchLocked() {
```

Comment:
This is a band-aid fix for a bug; the desired angle SHOULD be the locking angle. In many cases, it was not.